src/Mentat.UI/Controllers/CardController.cs

```diff
@@ -10,24 +9,24 @@ namespace Mentat.UI.Controllers
     public class CardController : Controller
     {
         private static Random random = new Random();
```

Comment:
For the refactor will you include removing this unused variable?

Reply:
Committed the changes. Resolving.